test/test_utils.cc

```diff
@@ -243,29 +243,29 @@ const char* const expected_device_enumeration_warnings_regex =
 
 namespace celerity::test_utils {
 
-detail::instruction_graph_generator::system_info make_system_info(const size_t num_devices, const bool supports_d2d_copies) {
+detail::system_info make_system_info(const size_t num_devices, const bool supports_d2d_copies) {
 	using namespace detail;
-	instruction_graph_generator::system_info info;
-	info.devices.resize(num_devices);
-	info.memories.resize(first_device_memory_id + num_devices);
-	info.memories[host_memory_id].copy_peers.set(host_memory_id);
-	info.memories[user_memory_id].copy_peers.set(user_memory_id);
-	info.memories[host_memory_id].copy_peers.set(user_memory_id);
-	info.memories[user_memory_id].copy_peers.set(host_memory_id);
+	system_info system;
+	system.devices.resize(num_devices);
+	system.memories.resize(first_device_memory_id + num_devices);
+	system.memories[host_memory_id].copy_peers.set(host_memory_id);
+	system.memories[user_memory_id].copy_peers.set(user_memory_id);
+	system.memories[host_memory_id].copy_peers.set(user_memory_id);
+	system.memories[user_memory_id].copy_peers.set(host_memory_id);
 	for(device_id did = 0; did < num_devices; ++did) {
-		info.devices[did].native_memory = first_device_memory_id + did;
+		system.devices[did].native_memory = first_device_memory_id + did;
 	}
-	for(memory_id mid = first_device_memory_id; mid < info.memories.size(); ++mid) {
-		info.memories[mid].copy_peers.set(mid);
-		info.memories[mid].copy_peers.set(host_memory_id);
-		info.memories[host_memory_id].copy_peers.set(mid);
+	for(memory_id mid = first_device_memory_id; mid < system.memories.size(); ++mid) {
+		system.memories[mid].copy_peers.set(mid);
+		system.memories[mid].copy_peers.set(host_memory_id);
+		system.memories[host_memory_id].copy_peers.set(mid);
 		if(supports_d2d_copies) {
-			for(memory_id peer = first_device_memory_id; peer < info.memories.size(); ++peer) {
-				info.memories[mid].copy_peers.set(peer);
+			for(memory_id peer = first_device_memory_id; peer < system.memories.size(); ++peer) {
+				system.memories[mid].copy_peers.set(peer);
 			}
 		}
 	}
-	return info;
+	return system;
```

Comment:
This local rename touches a lot of lines for no real reason.

Reply:
But it's not very hard to review, is it? :)

Reply:
But it pointlessly touches the history of those lines.

Reply:
I'm opinionated on this as you can tell, but not enough to delay cake allocation